src/components/common/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -41,6 +45,44 @@ class BaseCourseCard extends Component {
     return dropdownMenuItems;
   };
 
+  getStartDateMessage = () => {
+    const { pacing, startDate, endDate } = this.props;
+    let message = '';
+    if (pacing === 'self') {
+      const formattedEndDate = moment(endDate).format('MMMM D, YYYY');
+      message += `Complete at your own speed before ${formattedEndDate}.`;
```

Comment:
Should this just be `=`? and then we just `let message;` above? This caught my eye for whatever reason, but I defer to your style knowledge

Reply:
Ah I see, later it makes more sense. This is fine